examples/02_model_collaborative_filtering/multi_vae_deep_dive.ipynb

```diff
@@ -0,0 +1,1829 @@
+{
```

Comment:
<p>please change the weight path to a temporary directory:</p><pre class="ql-syntax" spellcheck="false">from tempfile import TemporaryDirectory

tmp_dir = TemporaryDirectory()
WEIGHTS_PATH = os.path.join(tmp_dir, "mvae_weights.hdf5")
</pre><p> </p><p>what is the meaning of the comments:</p><pre class="ql-syntax" spellcheck="false"># DEFAULT_SEED - Overfits
# 98765 - Overfits
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
when we use different SEED the models may overfit because this causes different training/validation/test sets.

Reply:
You should not get overfitting as long as the training, test , validation sets are separate (regardless of the value of the seed).